prime-router/settings/STLTs/VT/vt-doh.yml

```diff
@@ -0,0 +1,113 @@
+        suppressNonNPI: true
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "STANDARD"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: false
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: true
+          truncateHl7Fields:
+            - "OBX-23-1"
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'VT') or (Bundle.entry.resource.ofType(Patient).address.state = 'VT')"
+      qualityFilter:
+        #        - "Bundle.identifier.value.exists()"
+        #        - "Bundle.entry.resource.ofType(Patient).name.family.exists()"
+        #        - "Bundle.entry.resource.ofType(Patient).name.given.count() > 0"
+        #        - "Bundle.entry.resource.ofType(Patient).birthDate.exists()"
+        #        - "Bundle.entry.resource.ofType(Specimen).where(type.empty()).count() = 0"
+        #        - "Bundle.entry.resource.ofType(Patient).where(address.line[0].empty() and address.postalCode.empty() and telecom.where(system = 'phone').value.empty() and telecom.where(system = 'email').value.empty()).count() = 0"
+        #        - "((Bundle.entry.resource.ofType(Specimen).collection.collectedPeriod.exists() or Bundle.entry.resource.ofType(Specimen).collection.collected.exists())
+        #           or (Bundle.entry.resource.ofType(ServiceRequest).occurrence.exists() or Bundle.entry.resource.ofType(Observation).effective.exists()))"
+        # Message id is not empty (MSH-10)
+        - "Bundle.identifier.value.exists()"
+        - "Bundle.entry.resource.ofType(Patient).name.family.exists()"
+        - "Bundle.entry.resource.ofType(Patient).name.given.count() > 0"
+        - "Bundle.entry.resource.ofType(Patient).birthDate.exists()"
+        #  Specimen type (SPM-4) is not empty
+        - "Bundle.entry.resource.ofType(Specimen).where(type.empty()).count() = 0"
+        # Patient has at least a street address(PID-11-1), postal code (PID-11-5),phone(PID-13) or email(PID-13-4)
+        - "Bundle.entry.resource.ofType(Patient).where(address.line[0].empty() and address.postalCode.empty() and telecom.where(system = 'phone').value.empty() and telecom.where(system = 'email').value.empty()).count() = 0"
+        - "((Bundle.entry.resource.ofType(Specimen).collection.collectedPeriod.exists() or Bundle.entry.resource.ofType(Specimen).collection.collected.exists())
+                or (Bundle.entry.resource.ofType(ServiceRequest).occurrence.exists() or Bundle.entry.resource.ofType(Observation).effective.exists()))"
+      routingFilter: []
```

Comment:
This receiver does not use the default quality filters. They have their own set of filters that need to be moved over.
```
 "doesNotMatch(processing_mode_code,T,D)",
  "doesNotMatch(testing_lab_clia, 00Z0000015, 00Z0000016, 00Z0000042, 00Z0000043)",
  "hasAtLeastOneOf(order_test_date,specimen_collection_date_time,test_result_date)",
  "hasAtLeastOneOf(patient_street,patient_zip_code,patient_phone_number,patient_email)",
  "hasValidDataFor(message_id,equipment_model_name,specimen_type,test_result,patient_last_name,patient_first_name,patient_dob)",
  "isValidCLIA(testing_lab_clia,reporting_facility_clia)"
```

Reply:
Yep, sorry, commented them out to make it easier for me to test and then forgot to put them back in.